Field-Level Completeness Stats

OrdiNeu opened the pull request:
## Ticket(s)

- [DIG-1234](https://candig.atlassian.net/browse/DIG-1234)

## Description

- This PR adds field-level completeness stats to a new tab on the right

## Expected Behaviour

- The new tab properly displays completeness stats. To test completeness, try deleting a field from the `tests/clinical_ingest.json` before ingesting it.

## Related Issues (if appropriate)

-

## Screenshots (if appropriate)

### After PR

![image](https://github.com/CanDIG/candig-data-portal/assets/4656440/0c2bc2a6-ce9f-4283-85e4-210e112ed9eb)

## Types of Change(s)

-   [ ] 🪲 Bug fix (non-breaking change that fixes an issue)
-   [x] ✨ New feature (non-breaking change that adds functionality)
-   [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)

## Has it been tested for:

-   [ ] My change requires a change to the documentation
-   [ ] I have updated the documentation accordingly
-   [ ] Prettier linter doesn't return errors
-   [ ] Production branch PR browser testing: Chrome, Firefox, Edge, etc.
-   [x] Locally tested
-   [ ] Dev server tested
-   [ ] Production tested when merging into stable/production branch
-   [ ] [Runbook tasks](https://candig.atlassian.net/wiki/spaces/CA/pages/822018050/Frontend+Testing+Runbook) pass locally/on UHN-Dev
-   [ ] If visuals have changed, [Runbook](https://candig.atlassian.net/wiki/spaces/CA/pages/822018050/Frontend+Testing+Runbook) has been updated with new screenshots


[DIG-1234]: https://candig.atlassian.net/browse/DIG-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

OrdiNeu committed:
[wip] Completeness stats

OrdiNeu committed:
[wip]

OrdiNeu committed:
Work with the new genomic completeness stat

OrdiNeu committed:
Version update bugfixes

OrdiNeu committed:
[wip]

OrdiNeu committed:
Finish field-levle completeness graph

OrdiNeu committed:
Concatenate the site name before a cohort's

OrdiNeu committed:
Prettier fix

OrdiNeu committed:
Styling fixes

OrdiNeu committed:
Fixup chart direction

OrdiNeu committed:
Update the top menu icons so they don't all look the same

CourtneyGosselin reviewed (changes requested):
Currently there are issues with the display of the Clinical and Genomic completeness graphs across various pages, with half of them featuring mock data. Additionally, on the completeness page, the genomic graph requires a title change and is either displaying no data or appearing completely blank.

Furthermore, a change in font weight and size for the field-level card to match across all other cards on that page should be done. This will enhance the overall visual coherence and user experience.

OrdiNeu committed:
Fix genomic completeness, add to frontpage

- refactor completeness queries so that it's accessible from store/api.js

OrdiNeu committed:
Fix the title size for the field-level graph

OrdiNeu commented:
@CourtneyGosselin 
> Currently there are issues with the display of the Clinical and Genomic completeness graphs across various pages, with half of them featuring mock data. Additionally, on the completeness page, the genomic graph requires a title change and is either displaying no data or appearing completely blank.

Fixed the genomic graphs:
![image](https://github.com/CanDIG/candig-data-portal/assets/4656440/4d06db5b-3157-446a-a8c8-bc84a97d6433)


> Furthermore, a change in font weight and size for the field-level card to match across all other cards on that page should be done. This will enhance the overall visual coherence and user experience.

Fixed here: [7bf296a](https://github.com/CanDIG/candig-data-portal/pull/122/commits/7bf296a6e4230b8f261f97be1b86449068449c05)

CourtneyGosselin reviewed (changes requested):
Just need to remove the mock data from the data visualization page by either converting to real or removing completely.

OrdiNeu committed:
Remove completeness stats from the search page

CourtneyGosselin reviewed (approved):
LGTM
